cryptoki/src/mechanism/mod.rs

```diff
@@ -30,11 +30,17 @@ pub struct MechanismType {
 /// mechanism code is 0x80000170 with no parameters 
```

Comment:
Please update this line in the readme: https://github.com/stephaniegalang/rust-cryptoki/blob/ef70a3ed3fc2918a0b4c131fcf2507b14f1199c0/README.md?plain=1#L5

Reply:
done!